templates/wizard.php

```diff
@@ -63,7 +63,7 @@
 	and <a href="http://owncloud.org/promote"
 		target="_blank">spread the word</a>!')); ?>
 <?php else: ?>
-© 2014 <a href="https://owncloud.com" target="_blank">ownCloud Inc.</a>
+© 2015 <a href="https://owncloud.com" target="_blank">ownCloud Inc.</a>
```

Comment:
From a legal perspective it doesn´t make a difference. Maybe 2012-2015 looks nicer